src/coreclr/jit/codegencommon.cpp

```diff
@@ -1956,6 +1963,116 @@ void CodeGen::genGenerateMachineCode()
     /* The code is now complete and final; it should not change after this. */
 }
 
+//----------------------------------------------------------------------
+// genUpdateLiveRangesForTruncatedIGs -- find any instruction groups that have
+//   had their count altered and updated live ranges that refer to those groups
+//
+void CodeGen::genUpdateLiveRangesForTruncatedIGs()
+{
+#ifdef DEBUG
+    if (compiler->verbose)
+    {
+        printf("*************** In genUpdateLiveRangesForTruncatedIGs()\n");
+    }
+#endif
+    bool updatedGroup = false;
+
+    if (compiler->lvaCount > 0)
+    {
+        insGroup* ig = GetEmitter()->emitIGlist;
+        while (ig != nullptr)
+        {
+            if (ig->igFlags & IGF_UPD_ICOUNT)
+            {
+
+                unsigned int liveVarCount = varLiveKeeper->getVarCount();
+                if (liveVarCount > 0)
+                {
+#ifdef DEBUG
+                    if (compiler->verbose)
+                    {
+                        JITDUMP("IG%02u is marked with IGF_UPD_ICOUNT and has %d live variables\n", ig->igNum,
+                                liveVarCount);
+                    }
+#endif
```

Comment:
You don't need the `ifdef DEBUG` or `if (... verbose)` if using `JITDUMP`; it does that automatically. (You do need those things if you're calling `printf` directly)
```suggestion
                        JITDUMP("IG%02u is marked with IGF_UPD_ICOUNT and has %d live variables\n", ig->igNum,
                                liveVarCount);
```

Reply:
Cleaned up.